genai-on-vertex-ai/developer_productivity_with_genai/5_codey_talk_to_codebase_example.ipynb

```diff
@@ -1,3308 +1,3168 @@
 {
```

Comment:
<p>Would it make sense to update to latest version of Vertex AI SDK and langchain?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/applied-ai-engineering-samples/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
I just updated them to the versions that were fully tested. google-cloud-aiplatform==1.47.0 langchain==0.1.14 langchain-google-vertexai==0.1.3

The reason is that when I upgraded everything to the latest one, there were libraries conflicts.